src/usr/local/www/vpn_openvpn_client.php

```diff
@@ -638,6 +642,13 @@
 	    'This key is used to sign control channel packets with an HMAC signature for authentication when establishing the tunnel. ', '<br/>');
 
 	$section->addInput(new Form_Select(
+		'tlsauth_keydir',
+		'*TLS keydir direction',
+		$pconfig['tlsauth_keydir'],
+		openvpn_get_keydirlist()
+	))->setHelp('Leave this unchecked if you are not sure');
```

Comment:
It's not a checkbox so the note doesn't make sense here. Perhaps something like this instead:

"The TLS Key Direction must be set to complementary values on the client and server. For example, if the server is set to 0, the client must be set to 1. Both may be set to omit the direction, in which case the TLS Key will be used bidirectionally."